core/templates/dev/head/pages/exploration_editor/editor_tab/state_editor_responses.html

```diff
@@ -99,7 +99,7 @@
       </button>
     </div>
   </md-card>
-  <md-card style="margin: 20px 0px; padding: 0px;" ng-if="!isCurrentInteractionTrainable()">
+  <md-card style="margin: 20px 0px; padding: 0px;" ng-if="!isCurrentInteractionTrainable()"></md-card>
```

Comment:
The addition you've made here is not right. The closing md-card tag is on line 111.